adam_preprocessing/shape_stroke_extraction.py

```diff
@@ -927,6 +979,38 @@ def get_strokes(self):
         # FIXME: This return value is never used
         return objects
 
+    def combine_extractions(self, extractions: Sequence[ExtractionResult]):
+        num_obj = sum(e.num_obj for e in extractions)
+        prev_num_strokes = 0
+        prev_num_obj = 0
```

Comment:
`prev` sounds like it should be equal to the previous value, rather than the sum of all previous values. Maybe we want something like `num_strokes_seen` and `num_objs_seen`?